sections/advanced/refs.js

```diff
@@ -25,13 +25,17 @@ const Refs = () => md`
     border-radius: 3px;
   \`;
 
-  const Form = () => (
-    <Input
-      placeholder="Hover here..."
-      innerRef={x => this.input = x}
-      onMouseEnter={() => this.input.focus()}
-    />
-  );
+  class Form extends React.Component {
+    render() {
+      return (
+        <Input
+          placeholder="Hover here..."
+          innerRef={x => this.input = x}
```

Comment:
Thanks for fixing this up; while you’re add it can you add a function body around the assignment here, since that’s closer to prettier output. Sorry but a lot of people might feel it being off otherwise